eng/pipelines/templates/stages/vmr-build.yml

```diff
@@ -103,30 +103,30 @@ stages:
         useMonoRuntime: false              # 🚫
         withPreviousSDK: false             # 🚫
 
+    - template: ../jobs/vmr-build.yml
+      parameters:
+        # Changing the build name requires updating the referenced name in the source-build-sdk-diff-tests.yml pipeline
+        buildName: ${{ format('{0}_Online_CurrentSourceBuiltSdk', variables.centOSStreamName) }}
+        isBuiltFromVmr: ${{ parameters.isBuiltFromVmr }}
+        vmrBranch: ${{ variables.VmrBranch }}
+        targetArchitecture: x64
+        pool: ${{ parameters.pool_Linux }}
+        container:
+          name: ${{ variables.centOSStreamContainerName }}
+          image: ${{ variables.centOSStreamContainerImage }}
+        buildFromArchive: false            # 🚫
+        buildSourceOnly: true              # ✅
+        enablePoison: false                # 🚫
+        excludeOmniSharpTests: true        # ✅
+        runOnline: true                    # ✅
```

Comment:
We would get more value by having an offline leg in PR validation.  I know there was reasons this was configured to be online but I think those are somewhat mitigated by having this leg enabled in PR validation as it will eliminate the majority off offline regressions.  Thoughts on making this offline?

Reply:
Yep, makes sense to me. Fixed